brave/brave-core
# Remove special-case UI for no avalidable custodians

Resolves https://github.com/brave/brave-browser/issues/35615



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- With a clean profile, open the Rewards panel.
- Select a country with no supported providers (e.g. Antarctica) and enable Rewards.
- Close the Rewards panel.
- Open the Rewards panel.
- Verify that the normal connect messaging is displayed.
- Open the NTP.
- Verify that the normal connect messaging is displayed.
- Open the Rewards page.
- Verify that the normal connect messaging is displayed.
